env-files/account.template.env

```diff
@@ -110,3 +107,9 @@ API_BODY_JSON_LIMIT=1mb
 
 # Timeout in milliseconds to wait for a response as part of a request to an HTTP endpoint
 HTTP_RESPONSE_TIMEOUT_MS=3000
+
```

Comment:
I thought about putting these templates in the previous one, but CI was passing and I didn't want to add more files to it.